src/webview-contract/webviewDefinitions/kaito.ts

```diff
@@ -44,6 +44,17 @@ export type ToWebViewMsgDef = {
         workspace: WorkspaceCRD;
     }; // to webview after workspace CRD is generated
 };
+export type ToWebViewMsgDef2 = {
```

Comment:
you can add deploymentProgressUpdate to type ToWebViewMsgDef as well. I am wondering why we need another type ?

Reply:
This is actually outdated code, this has been removed in a further PR 👍